site/kubernetes.md

```diff
@@ -0,0 +1,87 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
+
+The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator).
+
+Here is an example command line for the Extract Domain Resource Tool:
+```
+<wls-deploy-home>/bin/extractDomainResource.sh -oracle_home /tmp/oracle	-domain_home /u01/mydomain -model_file /tmp/mymodel.yaml -domain_resource_file /tmp/operator/domain-resource.yaml
+```
+
+For the simplest case, the Extract Domain Resource Tool will create a sparse domain file. This is what is generated when there is not `kubernetes` section in the model, or that section is empty.
+```yaml
+apiVersion: weblogic.oracle/v6
+kind: Domain
+metadata:
+    name: DemoDomain
+spec:
+    domainHome: /u01/mydomain
+    imagePullSecrets:
+    -   name: '--FIX ME--'
+    webLogicCredentialsSecret: '--FIX ME--'
+    clusters:
```

Comment:
since "image" is a required attribute under spec, why doesn't the tool generate a placeholder for this?

Reply:
Added in latest commit, code changes covered in PR #521